routeros_api.class.php

```diff
@@ -92,7 +91,7 @@ public function encodeLength($length)
      *
      * @return boolean                If we are connected or not
      */
-    public function connect($ip, $login, $password)
+    public function connect($ip, $login, $password,$port)
```

Comment:
Shouldn't it have the default value for backward compatibility?

Reply:
You shouldn't have problems with previous versions.
It was only changed to allow connection through a port other than the default, without changing the code each time